### PR TITLE
Add recipe for pynput

### PR DESCRIPTION
**Pull request by kynan**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml" 
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/pynput/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - six
```
Comment (npavlovikj) [inline recipes/pynput/meta.yaml]: `python` should be added in the `run` section as well.

File: recipes/pynput/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
```
Comment (npavlovikj) [inline recipes/pynput/meta.yaml]: If with adding `python` in the `run` section you are still getting the same error, try adding `setuptools` and `sphinx >=1.3.1` in `host` too.

**Comment by synapticarbors** [comment]
There are a few outstanding issues:

- `setuptools-lint` is missing from the `defaults` and `conda-forge` channels
- The `COPYING` license file is not included in the source distribution. You may need to package it manually, but also ping the upstream creator https://conda-forge.org/docs/maintainer/adding_pkgs.html#packaging-the-license-manually

**Comment by ndmaxar** [comment]
setuptools-lint PR here: https://github.com/conda-forge/staged-recipes/pull/11207

**Comment by ndmaxar** [comment]
This should be closed in lieu of: https://github.com/conda-forge/staged-recipes/pull/11246

**Pull request closed without merging**